automation/get_target_servers.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/local/bin/python
+"""Script to get target server from Apigee"""
+
+import json
+import argparse
+import gzip
+import os
+from requests import Response
+from requests import Session
+
+from service import apigee_auth
+
+# Global session used for all requests.
+REQUEST = Session()
+APIGEE_API_URL = 'https://api.enterprise.apigee.com/v1/organizations/{}/environments/{}'
+
+
+def get_target_server(org_name: str, env_name: str, target_server_name: str):
```

Comment:
please move this method into the `service` folder.  Ideally you create a file called apigee_target_servers.py and put any methods related to the rest service there. 

Follow the other files to get an idea of the pattern.